src/components/ebay-listbox-button/index.marko

```diff
@@ -13,62 +12,89 @@ $ const {
     truncate,
     prefixLabel,
     prefixId,
-    unselectedText = '-',
+    unselectedText = "-",
     floatingLabel,
     collapseOnSelect,
     listSelection,
     variant,
     hasError,
+    split = "none",
     ariaDescribedby: describedby,
     ...htmlInput
 } = input;
-
 $ var selectedOption = (options as Option[])[state.selectedIndex];
 $ var selectedText = selectedOption && selectedOption.text;
+$ var selectedIcon = selectedOption && selectedOption.icon;
+$ var a11ySelectedIconText = selectedIcon && selectedOption && selectedOption.a11yIconText;
 $ var labelId = prefixId && component.getElId("label");
 $ var displayText = selectedText || unselectedText;
-$ var isForm = variant === 'form';
+$ var isForm = variant === "form";
+
+<macro|{ alwaysDisplay }: { alwaysDisplay: boolean }| name="render-text">
+    <if(selectedIcon)>
+        <span id=labelId class="btn__text">
+            <${selectedIcon}/>
+        </span>
+    </if>
+    <else-if(selectedText || alwaysDisplay)>
+        <span id=labelId class="btn__text">
+            ${displayText}
+        </span>
+    </else-if>
+</macro>
 
 <${truncate && !fluid ? "div" : "span"}
     ...processHtmlAttributes(htmlInput)
     key="container"
-    class=["listbox-button", inputClass, fluid && "listbox-button--fluid", isForm && `listbox-button--form`, hasError && 'listbox-button--error']
+    class=[
+        "listbox-button",
+        inputClass,
+        fluid && "listbox-button--fluid",
+        isForm && `listbox-button--form`,
+        hasError && "listbox-button--error",
+    ]
     on-expander-expand("handleExpand")
-    on-expander-collapse("handleCollapse")>
+    on-expander-collapse("handleCollapse")
+>
     <button
         key="button"
         class=[
             "listbox-button__control",
-            'btn',
-            borderless ? "btn--borderless" : 'btn--form',
+            "btn",
+            split !== "none" && `btn--split-${split}`,
+            borderless ? "btn--borderless" : "btn--form",
             truncate && "btn--truncated",
-            floatingLabel && "btn--floating-label"
+            floatingLabel && "btn--floating-label",
         ]
+        aria-label=a11ySelectedIconText
         value=selectedText
         type="button"
         disabled=disabled
         name=buttonName
         aria-haspopup="listbox"
         aria-describedby=describedby
-        aria-labelledby=(labelId && `${prefixId} ${labelId}`)
-        aria-invalid=(invalid && "true")>
+        aria-labelledby=labelId && `${prefixId} ${labelId}`
```

Comment:
Can't have aria-labelledby _and_ aria-label. It's one or the other.

Reply:
Actually, just notice this is on listbox-button, not phone-input itself. So long as aria-label always takes precedence over aria-labelledby in listbox-button we should be good.